js/app.js

```diff
@@ -0,0 +1,487 @@
+    if((bet/2) === 100){
+      newChip.src = "../images/100Chip.png"
+    }else if(bet/2 === 500){
+      newChip.src = "../images/500Chip.png"
+    }else if(bet/2 === 1000){
+      newChip.src = "../images/1000Chip.png"
+    }
+  let newChip2 = document.createElement('img')
+    newChip2.setAttribute("class", 'chip')
+    if((bet/2) === 100){
+      newChip2.src = "../images/100Chip.png"
+    }else if(bet/2 === 500){
+      newChip2.src = "../images/500Chip.png"
+    }else if(bet/2 === 1000){
+      newChip2.src = "../images/1000Chip.png"
+    }
+    betArea.appendChild(newChip)
+    betArea.appendChild(newChip2)
+    chip.play()
+    
+    chip.play()
+    
+}
+
+function betPayout(bet){
+  if(outcome === "W"){
+    score = score + bet
+    clap.play()
+    if(clap.currentTime > 4){
+      clap.pause()
+    }
+  }
+  if(outcome === "L"){
+    score = score - bet
+  }
+  if(outcome === "BJ"){
+    score = score + (bet * 1.5)
+    clap.play()
+  }
+  if(outcome === "T"){
+    score = score + 0
+  }
+}
+
+function p1TotalRender(){
+  playerTotal = totalHolder
+  player1TotalElem.removeAttribute("class") //removes color text
+  if(playerTotal === "Bust"){
+    player1TotalElem.classList.add("redText")
+  }
+  if(playerTotal < 21){
+    player1TotalElem.classList.add("greenText")
+  }
+  if(playerTotal === 21){
+    player1TotalElem.classList.add("gold-gradient-text")
+  }
+  player1TotalElem.textContent = `( ${playerTotal} )` //player total render
+}
+function dTotalRender(){
+  dealerTotal = totalHolder
+  dealerTotalElem.removeAttribute("class") //removes color text
+  if(dealerTotal === "Bust"){
+    player1TotalElem.classList.add("redText")
+  }
+  dealerTotalElem.textContent = `( ${dealerTotal} )`
+}
+
+function p1Render(){
+  playerHandElem.innerHTML= ""
+  playerHand.forEach((card, idx)=>{
+  renderCard(card, idx, playerHandElem)
+})
+}
+function dRender(){
+  dealerHandElem.innerHTML= ""
+  dealerHand.forEach((card, idx)=>{
+    renderCard(card, idx, dealerHandElem)
+  })
+}
+
+function renderCard(cardPicked, idx, location){
+  let newCard = document.createElement("div")
+  newCard.setAttribute("class", `card ${cardPicked}`)
+  //works as I thought it would it animates the entire hand LOL 
+  //newCard.classList.add("animate__animated", "animate__slideInDown")
+  location.appendChild(newCard)
+}
+
+function shuffleDeck(){
+  console.log("Shuffled deck")
+  blackjackDeck = [...sixDeckOrig]
+}
+
+// function messageColor(){
+//   if(outcome = "W"){
+//     playGame.classList.add("red-gradient-text")
+//   }else if(outcome = "L"){
+//     playGame.classList.add("redText")
+//   }else if(outcome = "BJ"){
+//     playGame.classList.add("gold-gradient-text")
+//   }else if(outcome = "T"){
+//     playGame.removeAttribute('class')
+//   }
+// }
+
+//function to celebrate achievement...
+function congrat(){
+    playGame.textContent="Broke the House, Congrats"
+    tada.play()
+    confetti.start(2000)
+}
+//how did it come to this 😭
```

Comment:
Lol (also LOVE that you can take out a loan to continue gambling, extremely chaotic energy.)